target/bin/getmail-cron

```diff
@@ -1,8 +0,0 @@
-#! /bin/bash
-
-GETMAILDIR=/tmp/docker-mailserver/getmail
-for FILE in /etc/getmailrc.d/getmailrc*; do
-  if ! pgrep -f "${FILE}$" &>/dev/null; then
```

Comment:
I could swear, that we already discussed this? Or maybe I am starting to lose the overview 😆 

> I have no idea what the pgrep here was for.. ensuring there wasn't a getmail process already running for that specific --rcfile arg I guess?

Correct.

>  there's the possibility of --rcfile args being too many, IIRC there is a limit in length bash supports regarding something like that)

Correct.

Reply:
> I could swear, that we already discussed this?

We discussed the multiple `--rcfile` to single `getmail` call on another review comment that I brought up the same topic. No worries :)

This one was about the `pgrep` usage, there was nothing on the original PR that explained what it was needed for.

The 2nd part was about `process_check_restart.bats`, which could include a check for the service wrapper you introduce with this PR, rather than non-daemonized actual `getmail` process(es)... but for our own service scripts I don't know if there is as much value with the sanity check that test is for? 🤷‍♂️ Up to you.

Reply:
I've added getmail to the check 👍